frontend/src/components/common/RenderDateTime.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
```

Comment:
This is a new file?

Reply:
Yup, we only had RenderDate, so it's a separate component for DateTime.

Reply:
Gotcha. I didn't realize that this was slightly different from RenderDate. It looks like this is the difference?
```javascript
      showTime={{ format: "HH:mm" }}
      format="YYYY-MM-DD HH:mm"
      style={{ width: "100%" }}
```

My inclination would be to pass in `format` in the props and default to `YYYY-MM-DD` if it's not provided. Or something like a `minutes` boolean to check whether or not to show minutes.

Is there a reason why you preferred to make a new component instead of adding an item to props?

Reply:
There is some  minuta about the props that we should deal with centrally instead of having everyone figure out... i'm not opposed to just flagging, but it was the easiest way to debug and get it started.

Reply:
Not sure what you mean by dealing with props centrally. They're only relevant to a specific component. Maybe I'm misunderstanding you?

Anyway, if you default to the existing behaviour and just add the `minutes` boolean/flag then no consumers of this component are effected. That's my personal preference here, given that it's almost entirely copy/paste of the `RenderDate` component and the different caused by the minutes flag is quite minimal.

Reply:
Not sure this is the way you would do it, so let me know if the flag is ok.